tasks/04_helper_scripts.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Install helper scripts
```

Comment:
I couldn't get the `never` tag to work properly here and wouldn't have the time to review the safety of those shell scripts if more are added. So removed this for now. You can still add those scripts in your own playbook. Or maybe start a repo with `borgmatic-helpers`. There is already a similar thing for Prometheus.